### PR TITLE
docker-worker deps update

### PR DESCRIPTION
**Pull request by djmitche**
Mostly, this updates lodash

### CODE REVIEW

File: workers/docker-worker/package.json [hunk 1]
```diff
@@ -80,7 +80,6 @@
     "ws": "^7.1.2"
   },
   "devDependencies": {
-    "github": "^13.1.0",
```
Comment (helfi92) [inline workers/docker-worker/package.json:83]: Why was this removed?

Comment (djmitche) [reply workers/docker-worker/package.json:83]: It's unused -- never require'd.  And it's an ancient version, which was pulling in old deps.
